bootstrap.sh

```diff
@@ -224,14 +224,30 @@ fi
 echo "Installing dependencies for building web UI"
 angular_cli_dir=$VTROOT/dist/angular-cli
 web_dir2=$VTTOP/web/vtctld2
-rm -rf $angular_cli_dir
-cd $VTROOT/dist && git clone https://github.com/angular/angular-cli.git --quiet
-cd $angular_cli_dir && git checkout 3dcd49bc625db36dd9f539cf9ce2492107e0258c --quiet
+
+if [[ -d $angular_cli_dir ]]; then
+  echo "skipping angular cli download. remove $angular_cli_dir to force download."
```

Comment:
We're going to be needing to update the particular git hash for angular-cli often, since it's still changing quickly. Can you make this skip only if the dir exists and the version matches what we want? Ideally we should be able to update the desired version in bootstrap.sh, and then just tell everyone to re-run bootstrap.sh.